library/src/main/java/in/krharsh17/barview/BarGroup.java

```diff
@@ -136,7 +140,10 @@ void setupValue() {
         value.setPadding(0, 0, 0, dp(8));
         value.setTextColor(Color.parseColor(VALUE_TEXT_COLOR));
         value.setTextSize(TypedValue.COMPLEX_UNIT_SP, VALUE_FONT_SIZE);
-        value.setTypeface(Typeface.createFromAsset(context.getAssets(), "fonts/josefin_sans.ttf"));
+        if(VALUE_FONT!=null)
+            value.setTypeface(Typeface.createFromAsset(context.getAssets(), VALUE_FONT));
+        else
```

Comment:
Please remove the `else` case, default font to be used should be the system font